pixartjquery.js

```diff
@@ -0,0 +1,23 @@
+$(document).on("ready", function(){
+  var form = $("#form")
+  var brush = $(".brush")
+  var input = $("#color-field")
+  var button = $("#set-color")
+
+$("#form").on("submit", function(){
+  event.preventDefault();
+  console.log("happy")
+  brush.css("backgroundColor", input.val());
+});
+
+
+for(x = 0; x < 10000; x++) {
+  var div = $("<div>", {class: "square"});
+  $("body").append(div);
+
+};
+
+$(".square").on('mouseover', function() {
+$(this).css("backgroundColor", input.val());
```

Comment:
What you have here works fine, but it might be better to pull the color from the background color of the brush instead of  from the input. Also be careful with you indenting, proper indentation is super important in making code readable! Generally a good rule is any blocks of code in between curly braces should be indented (like in your for loop). Going by that rule everything inside the `$(document).ready(function() {` should be indented at least one level.